resources/disk/make_rootfs.sh

```diff
@@ -37,12 +39,14 @@ Options:
   -d debdir         Directory containing .deb packages for the Linux image.
   -w workdir        Working directory for the kernel build.
   -o diskfile       Name of the resulting disk file.
-  -f flavour        The Ubuntu flavour desired. Defaults to "focal"
+  -f flavour        The Ubuntu flavour desired. Defaults to "focal".
+  -f disk size      The size of the resulting rootfs.
```

Comment:
We should also add a comment about what is the size unit (or just give a few examples so people know how to use it).

Reply:
Done.